### PR TITLE
Typo fix and readability+new command update to "help" list

### PR DESCRIPTION
**Pull request by erenbektas**
Relevant issue: Fixes https://github.com/GrafeasGroup/blossom/issues/426

## Description:

1. Fixed the typo where command to get the check percentage in the help menu was "watchlist" instead of "watchstatus".
2. Readability update: Commands first and descriptions after.
3. Addition to the help menu: "\@Blossom checkstats <username>"

## Screenshots:

{if applicable}

## Testing Instructions:

{testing instructions}

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [ ] Tests (if applicable)
- [ ] Success Criteria Met
- [x] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

**Comment by itsthejoker** [comment]
Does swapping the order of the help menu improve readability? When I look for something, I know the action but not the command, so it would stand to reason that it should go "action: command". Is there a specific reason you set it to "command: action"?

**Comment by erenbektas** [comment]
> Does swapping the order of the help menu improve readability? When I look for something, I know the action but not the command, so it would stand to reason that it should go "action: command". Is there a specific reason you set it to "command: action"?

Uhh actually Tim told about this on slack and nobody said otherwise. That's why I also implemented it

**Comment by MurdoMaclachlan** [comment]
As a reader "command: action" works better for me too. I like having the consistent start, almost reminiscent of a bullet point. Also looks a bit neater this way, in my opinion.

**Comment by seeroflights** [comment]
This is a nit but can we also alphabetize the list while we're here?

**Comment by erenbektas** [comment]
QAkright then, let's decide on those things and then I can create another pr instead of this one.

+1 to alphabetical list
+1 to command: action

File: blossom/strings/en_US.toml [hunk 1]
```diff
@@ -35,17 +35,18 @@ messages = [
...
-Get the warnings of a given user: `@Blossom warnings <username>`
-Render this message: `@Blossom help`
+`@Blossom info <username>`: Get information about a user.
+`@Blossom info`: Get general server information.
+`@Blossom blacklist <username>`: Blacklist / unblacklist user.
+`@Blossom reset <username>`: Toggle CoC status of user.
+`@Blossom watch <username> <percentage>`, where `<percentage>` is a number between 0 and 100: Overwrite check percentage.
+`@Blossom unwatch <username>`: Reset check percentage.
+`@Blossom watchstatus <username>`: Get the check percentage for a user.
+`@Blossom watchlist <sorting>`, where `<sorting>` is either "percentage" (default) or "alphabetical".: List all overwritten check percentages.
```
Comment (TimJentzsch) [inline blossom/strings/en_US.toml:45]: Nit: I'd remove that full stop
```suggestion
`@Blossom watchlist <sorting>`, where `<sorting>` is either "percentage" (default) or "alphabetical": List all overwritten check percentages.
```

**Review by TimJentzsch — CHANGES_REQUESTED** [review]
Looks much better already!

Let's do the alphabetical ordering as requested by QA and then this should be good to go :)

**Review by TimJentzsch — APPROVED** [review]

**Pull request merged**